src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        int randomNumber = new Random().nextInt(100);
+        ball.setNumber(randomNumber);
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
ColorSupplier should be `private final` class-level variable

Reply:
updated